.github/workflows/nightly_release.yml

```diff
@@ -59,36 +61,32 @@ jobs:
           cd v-analyzer
           v run build.vsh release
 
-      - name: Create artifact
+      - name: Upload artifact
         uses: actions/upload-artifact@v3
         with:
-          name: v-analyzer-${{ matrix.target }}
+          name: ${{ env.ARTIFACT }}
           path: ./v-analyzer/bin/v-analyzer${{ matrix.bin_ext }}
 
-  nightly-release:
-    if: github.event.repository.full_name == 'v-analyzer/v-analyzer'
-    name: Create Nightly GitHub Release
-    needs: dist
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/download-artifact@v3
-
-      - name: 'Create zip'
+      - name: Prepare release
         shell: bash
         run: |
-          chmod 755 v-analyzer-*/*
```

Comment:
That was needed, otherwise the zips will contain binaries that are not marked as executable.

Reply:
Or is the idea to make a .zip before the upload, on the same job that created the executable?

Reply:
Yes, correct about the job. The binaries in the linked release on the fork can be checked how they come out.

Reply:
How it is done here it's not required to first upload the bins as artifacts and downloading them again when they should be uploaded as release binaries.

Reply:
![Screenshot_20231215_122421](https://github.com/v-analyzer/v-analyzer/assets/34311583/d8e89d33-a6e6-4cc8-a59c-95ea3ad60eaa)

Reply:
Nice. It is both simpler, and more functional that way.